### DISCUSSION TITLE
Where are the stats coming from?

### DISCUSSION THREAD

**Discussion by SteveTaskersSon** [110yards/fantasy#243] [General]
Just curious, where is the site pulling its stats from? We've noticed a few inconsistencies between 110 and game tracker stats, as well as the stat sheet that is released after the games.  Please don't take this question as a criticism or complaint as it is not.  I think everyone is aware of the chaos the cfl has created with switching its stats system at the last minute, and we're definitely aware and very appreciative of all the time and hard work you have put into getting this site going again!!!

**Comment by mdryden** [comment]
Hi Steve,

The stats are pulled together from multiple sources.  I am currently evaluating other options, though I suspect we will be stuck with what we have for the remainder of the season.

I'm also considering some moderation tools for next year (perhaps late this year) which could allow a trusted set of users to make stat corrections.

**Reply by SteveTaskersSon** [reply]
Thanks for the response!  Thanks again for keeping this site going, it’s way better than what the CFL offers and we all greatly appreciate the long hours you’ve put into it.
